packages/js/src/components/modals/PremiumSEOAnalysisUpsell.js

```diff
@@ -28,7 +32,7 @@ const PremiumSEOAnalysisUpsell = ( props ) => {
 	return (
 		<UpsellBox
 			title={ __( "Get more help with writing content that ranks", "wordpress-seo" ) }
-			description={ __( "Check your text on even more SEO criteria and get an enhanced keyphrase analysis, making it easier to optimize your content.", "wordpress-seo" ) }
+			description={ props.description }
```

Comment:
Making the description as a prop so we can reuse it for the highlighting upsell modal in Elementor